migrate/src/main/scala/migrate/Main.scala

```diff
@@ -115,7 +119,7 @@ object Main {
       .withClasspath(classpath.paths.map(_.toNio).asJava)
       .withScalacOptions(compilerOptions.asJava) // not sure which compiler option we need here !!
```

Comment:
We want all Scala 2 compiler options of the project because Scalafix wants to call the Scala 2 compiler. Otherwise it could fails.

Reply:
call it only if the semantic files are not found